### PR TITLE
yngshan/new_OTC_index

### PR DESCRIPTION
**Pull request by yngshan**

### CODE REVIEW

File: src/templates/haml/get_started/otc_indices.html.haml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+      .section
+        %a#what-are-the-otc-indices
+        .section-content
+          #section-1
+            .grd-grid-12
+              %h1= l('What are OTC indices?')
+            .grd-grid-8.grd-grid-mobile-12
+              %p= l('[_1]\'s OTC Indices closely tracks the corresponding Cash Index*.', stash->{website_name});
+              %p= l(' With real-time charts and prices, these indices provide more exciting opportunities to trade on the financial markets.');
+              %p= l('*Cash index here denotes the market indices that are presently offer on [_1]. For example the German OTC Index closely tracks the German Index.', stash->{website_name});
```
Comment (mm-binary) [inline src/templates/haml/get_started/otc_indices.html.haml]: offered?

File: src/templates/haml/get_started/otc_indices.html.haml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+            %a{:href => "#what-are-the-otc-indices"}= l('What are the OTC Indices?')
+    .grd-grid-9.grd-grid-mobile-12.grd-parent
+      .section
+        %a#what-are-the-otc-indices
+        .section-content
+          #section-1
+            .grd-grid-12
+              %h1= l('What are OTC indices?')
+            .grd-grid-8.grd-grid-mobile-12
+              %p= l('[_1]\'s OTC Indices closely tracks the corresponding Cash Index*.', stash->{website_name});
```
Comment (mm-binary) [inline src/templates/haml/get_started/otc_indices.html.haml]: track?

File: src/templates/haml/get_started/otc_indices.html.haml (hunk 1 above)
Comment (yngshan) [inline src/templates/haml/get_started/otc_indices.html.haml]: This template still pending latest version from Nikki ...you can ignore first :)

**Comment by jy-binary** [comment]
can you add me as a collaborator to your binary-static-www2 fork so I can see the "Update branch" button?

**Comment by yngshan** [comment]
@jy-binary ok added

**Comment by jy-binary** [comment]
pls check conflicts

**Pull request merged**